extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -137,27 +135,11 @@ FrameworkEndpointsBuildItem frameworkEndpoints(NonApplicationRootPathBuildItem n
         Set<String> frameworkEndpoints = new HashSet<>();
         for (RouteBuildItem route : routes) {
             if (FRAMEWORK_ROUTE.equals(route.getRouteType())) {
-                if (route.getConfiguredPathInfo() != null) {
-                    String endpointPath = route.getConfiguredPathInfo().getEndpointPath(nonApplicationRootPath,
-                            managementBuildTimeConfig, launchModeBuildItem);
-                    frameworkEndpoints.add(endpointPath);
+                if (route.getAbsolutePath() == null) {
```

Comment:
For one of the route, it's null, not exactly sure which it is, maybe the 404 one.